### PR TITLE
doc: Add guide for Daizy cloud integration

### PR DESCRIPTION
**Pull request by revbingo**
#### Summary

Adds documentation for the Daizy cloud integration recently merged to the webhooks repo

#### Screenshots

image

#### Changes

- Adds page for Daizy cloud integration
#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [x] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/cloud-integrations/daizy/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Use [Daizy](https://daizy.io) to rapidly build, deploy and scale Internet of Things applications and stay in control of your data.
+
+The Daizy platform provides an intuitive management system for IoT devices and projects, and delivers data directly to any existing or future business systems. Daizy enables rapid scaling by reducing time to market for any IoT implementation while minimizing cost, improving flexibility and maintaining security.
+
+<!--more-->
+
+## Prerequisites
+
+1. A user account and organisation created within the [Daizy platform](https://portal.daizy.io/login).
+2. LoRa devices enrolled and deployed in your project.
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/daizy/index.md:16]: This is redundant, we assume devices are already sending data to TTS for all integrations

File: doc/content/integrations/cloud-integrations/daizy/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Set up Daizy
+
+The details required for creating the webhook can be found under Compose > Connectivity > IoT Endpoint
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/daizy/index.md:20]:
```suggestion
The details required for creating the webhook on Daizy can be found under **Compose &#8594; Connectivity &#8594; IoT Endpoint**.
```

File: doc/content/integrations/cloud-integrations/daizy/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+{{< figure src="connectivity.png" alt="Daizy Connectivity tab" >}}
+
+Copy the values for the Organisation Slug, and Uplink API Key.
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/daizy/index.md:24]:
```suggestion
Copy the **Organisation slug** and **Uplink API key** values.
```

File: doc/content/integrations/cloud-integrations/daizy/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Configure {{% tts %}}
+
+In {{% tts %}}, create a webhook using the ready made **Daizy** [Webhook template]({{< ref "/integrations/webhooks/webhook-templates" >}}).
+
+Give your Webhook a name in the **Webhook ID** field.
+
+Copy the **Organisation slug** and **Uplink API Key** from your **IoT Endpoint** details screen in Daizy, and fill these in accordingly.
```
Comment (nejraselimovic) [inline doc/content/integrations/cloud-integrations/daizy/index.md:32]:
```suggestion
Paste the **Organisation slug** and **Uplink API key** values you copied in the previous steps to the corresponding fields of the Webhook template.
```

**Review by nejraselimovic — APPROVED** [review]
Thanks for your contribution @revbingo! This looks great, I just added a few points to polish, and we can merge this after.

File: doc/content/integrations/cloud-integrations/daizy/index.md (hunk 1 above)
Comment (revbingo) [reply doc/content/integrations/cloud-integrations/daizy/index.md:16]: The intent here was that the user needs the devices enrolled in their Daizy project, not in TTS, I will make this clear.

**Pull request merged**